### DISCUSSION TITLE
<Link prefetch={false}> and onMouseEnter

### DISCUSSION THREAD

**Discussion by nonamich** [vercel/next.js#40768] [Help]
✌
And yet, how do I disable prefetch for "onMouseEnter"?

**Comment by wwwdepot** [comment]
Create your own component and use `router.push(href)`.

See example: 

https://github.com/vercel/next.js/blob/11dd1de655bfdab82d96b7bfe76c6fa1ef70337d/examples/using-router/components/Header.js#L12-L33
